controllers/podsweep_runner.go

```diff
@@ -87,6 +89,14 @@ func (r *PodSweeper) run(ctx context.Context) error {
 
 		v, ok := po.Annotations[constants.PodDeletionTimeKey]
```

Comment:
Why are you still using annotations?

Reply:
The deletion time annotation is still in use.
When a user requests to extend a failed pod from Slack, the slack-agent will give an annotation.
So pod-sweeper still prioritizes annotation, and if there is no annotation, pod-sweeper uses the API.

Reply:
Is it not possible for slack-agent to set the deletion time through the API?
It is better not to use multiple means to achieve one purpose.

Reply:
Indeed, you are right.
I'll think about a fix to update the delete time from slack-agent via API.

Reply:
I will deal with this matter in another PR.